data/scripts.js

```diff
@@ -897,7 +897,7 @@ exports.BattleScripts = {
 	},
 
 	runMegaEvo: function (pokemon) {
-		const isUltraBurst = !pokemon.canMegaEvo;
+		const effectType = !pokemon.canMegaEvo ? '-burst' : '-mega';
```

Comment:
Double negative might be confusing? I'd prefer `pokemon.canMegaEvo ? '-mega' : '-burst'`.